testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -318,6 +333,12 @@ def _initObservationConfigurationParameters(cls, parser, defaults):
     parser.add_argument(
         '--aws_profile', default=defaults.get('AWS_PROFILE', None),
         help='aws command-line tool --profile parameter when observing AWS.')
+    
+    parser.add_argument(
+        '--appengine_credentials_path',
```

Comment:
Mention the default here is to use --gce_credentials_path whose default is application credentials.

I'm not sure how you'd use application credentials for appengine but an explicit service account for GCE.